src/vs/workbench/api/common/extHostLanguageFeatures.ts

```diff
@@ -423,6 +424,13 @@ class CodeActionAdapter {
 			if (!candidate) {
 				continue;
 			}
+
+			const currentKind = (candidate as vscode.CodeAction).kind;
+			const nbKind = new CodeActionKind('notebook');
+			if (codeActionContext.triggerKind !== vscode.CodeActionTriggerKind.Invoke && currentKind && nbKind.contains(currentKind)) {
```

Comment:
I believe You should use the actual implementing types here (extHostTypes) instead of those from `vscode`